examples/text_classification-ort.ipynb

```diff
@@ -0,0 +1,1289 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "75bcf2ee-3fb0-4c3a-94dc-e0fc867099b2",
+   "metadata": {},
+   "source": [
+    "# How to fine-tune a distilbert model with ONNX Runtime\n",
+    "\n",
+    "This notebook is largely inspired by the text classification [notebook of Transformers](https://github.com/huggingface/notebooks/blob/main/examples/text_classification.ipynb) which takes PyTorch as backend for fine tuning. \n",
+    "\n",
+    "Here, instead of `Trainer`, you will use the `ORTTrainer` class in [🏎️ Optimum ](https://github.com/huggingface/optimum) library and take [ONNX Runtime](https://microsoft.github.io/onnxruntime/) as backend to accelerate the training. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9408e05-175b-4573-8314-9ae4ba7d3453",
+   "metadata": {},
+   "source": [
+    "__Dependencies__\n",
+    "\n",
+    "To use ONNX Runtime for training, you need a machine with at least one NVIDIA or AMD GPU.\n",
+    "\n",
+    "CUDA and cuDNN need to be properly installed, before the installation of ONNX Runtime training module. Please follow the instruction in [Optimum's documentation](https://huggingface.co/docs/optimum/onnxruntime/trainer) to set up your environment.\n",
```

Comment:
https://huggingface.co/docs/optimum/onnxruntime/trainer currently does not give info on how to setup the environment.

Reply:
Hi @fxmarty,

That's true, the documentation with detailed setup steps will be updated in [PR#376](https://moon-ci-docs.huggingface.co/docs/optimum/pr_376/en/onnxruntime/trainer#prerequisite). I temporarily put the link to the main doc, this shall be replaced after the doc makes its way to the main of optimum, thanks for noticing.

Reply:
Ok great I missed it!